nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoCorsCsrfFilter.java

```diff
@@ -95,13 +105,57 @@
 
     public static final String ALLOW_NULL_ORIGIN_DEFAULT = "false";
 
+    /** @since 10.3 */
+    public static final String CSRF_TOKEN_NS_PROP = "nuxeo.csrf.token";
+
+    /**
+     * Allows to enforce the use of a CSRF token. Under the {@value #CSRF_TOKEN_NS_PROP} namespace.
```

Comment:
Allows enforcing the use of a CSRF token, under the {@value #CSRF_TOKEN_NS_PROP} namespace.